scripts/migrate.js

```diff
@@ -0,0 +1,45 @@
+const { migrate } = require('postgres-migrations');
+const path = require('path');
+
+const { NODE_ENV } = process.env;
+
+// This code is repeated a few times in our scripts. Can you factor it out into a shared helper function?
```

Comment:
Copy pasted file again. Regards comments like this, they are intended to help you when working through the solution and are not meant to be left in the repo